osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -268,21 +269,23 @@ private Drawable createDrawablePerformance()
                 Direction = FillDirection.Horizontal,
                 Children = new[]
                 {
-                    new OsuSpriteText
+                    new SpriteTextWithTooltip
                     {
                         Anchor = Anchor.BottomLeft,
                         Origin = Anchor.BottomLeft,
                         Font = font,
-                        Text = $"{Score.PP:0}",
-                        Colour = colourProvider.Highlight1
+                        Text = Score.PP.ToLocalisableString(@"N0"),
+                        TooltipText = Score.PP.ToLocalisableString(@"N1"),
```

Comment:
redundant decimal digits are back

<img width="2114" height="126" alt="Image" src="https://github.com/user-attachments/assets/59b280a2-5868-4605-8aef-eb47196ff74c" />

Reply:
I intentionally brought them back because it's less verbose with a single digit and it further signifies that the number can be a decimal, rather than making the tooltip text exactly match the displayed number.